### PR TITLE
Bumping mortgage calculator

### PR DESCRIPTION
**Pull request by slaywell**
Bumping version of Mortgage Calculator. <strike>This has also updated dough-ruby, minitest, rake and underscore-rails.</strike>

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -470,9 +470,9 @@ GEM
     method_source (0.8.2)
     mime-types (1.25.1)
     mini_portile (0.6.2)
-    minitest (5.8.3)
+    minitest (5.8.4)
     modernizr-rails (2.6.3)
-    mortgage_calculator (1.3.8.631)
+    mortgage_calculator (1.3.8.633)
```
Comment (tomas-stefano) [inline Gemfile.lock]: I would just commit this line.

**Comment by slaywell** [comment]
Thanks @tomas-stefano @munckymagik, it's just the one change now.

File: Gemfile.lock [hunk 2]
```diff
@@ -806,3 +806,6 @@ DEPENDENCIES
   validate_url (= 1.0.0)
   vcr
   webmock
+
+BUNDLED WITH
+   1.11.2
```
Comment (johnplayer1982) [inline Gemfile.lock]: Hey @slaywell I dont think you need this bit either

**Comment by Tr4pSt3R** [comment]
:+1:

**Comment by slaywell** [comment]
@johnplayer1982 Removed the bundle info now.

**Comment by munckymagik** [comment]
:+1:

**Comment by johnplayer1982** [comment]
:+1:

**Comment by tomas-stefano** [comment]
:+1:

**Pull request merged**